documentation/modules/con-kafka-connect-authentication.adoc

```diff
@@ -63,27 +65,41 @@ spec:
   # ...
   authentication:
     type: scram-sha-512
-    username: my-connect-user
+    username: my-connect-username <1>
     passwordSecret:
-      secretName: my-connect-user
-      password: my-connect-password-key
+      secretName: my-connect-secret-name <2>
+      password: my-connect-password-field <3>
   # ...
 ----
 
-== SASL based PLAIN authentication
+<1> `username`: The username.
+<2> `secretName`: The name of the Secret that contains the username and password.
+<3> `password`: The name of the field in the Secret under which the password is stored.
+
+[WARNING]
+====
+Do NOT specify the actual password in the `passwordSecret.password` property.
+====
+
+To use this method, see xref:proc-configuring-kafka-connect-authentication-scram-sha-512-{context}[].
+
+== SASL-based PLAIN authentication
```

Comment:
Same here. I think we can just link to the appendix description https://strimzi.io/docs/operators/master/using.html#type-KafkaClientAuthenticationPlain-reference. Modify with any changes from this update